python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -169,3 +182,10 @@ def tearDown(self):
         stop_ganga()
         import sys
         sys.stdout.flush()
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.wipe_repo is True:
+            import shutil
+            shutil.rmtree(cls.gangadir, ignore_errors=True)
```

Comment:
Well in that case I'm happy to drop the method in this case, the main change I wanted was in setUp anyway.
Unless you can devise a fix, I'm assuming ```self.__class__.gangadir = something``` should work well enough?